Classes/Events/Listener/GenerateRequestTokenListener.php

```diff
@@ -35,10 +35,9 @@ public function __invoke(BeforeRequestTokenProcessedEvent $event): void
         // wenn oauth cookie => get time from token from cookie
         // wenn request within 5 sec => generate and set token
 
-        // $requestToken = RequestToken::create('core/user-auth/be');
-        // $event->setRequestToken($requestToken);
+        $requestToken = RequestToken::create('core/user-auth/be');
```

Comment:
This needs to also be adjusted for frontend.

```suggestion
        $user = $event->getUser();
        $requestToken = RequestToken::create('core/user-auth/' . strtolower($user->loginType));
```